src/pages/steps/AdditionalInformationStep/OrganizerPicker.tsx

```diff
@@ -249,12 +256,14 @@ const OrganizerPicker = ({
               </Inline>
             </Stack>
           ) : (
-            <Inline>
+            <Inline width="100%" flexWrap="wrap">
               <RecentUsedOrganizers
                 organizers={recentUsedOrganizers}
                 onChange={handleSelectRecentOrganizer}
+                width={isSmallView ? '100%' : '60%'}
+                maxWidth="50rem"
               />
-              <Stack>
+              <Stack width={isSmallView ? '100%' : '40%'}>
```

Comment:
```suggestion
              <Stack width={{ s: '100%', default: '40%' }}>
```

You should be able to write it like this